include/client.h

```diff
@@ -70,54 +70,28 @@ class Client
 
     Status queryAccount(id_t accountId, Account& account);
 
-    Status queryInstrumentsList(id_t accountId, Instruments& instruments);
+    Status queryFeedsList(id_t accountId, Feeds& feeds);
 
-    // not sure we can get instrument by ID, as there is no such member for now
-//    Status queryInstrument(id_t accountId, id_t instrumentId, Instrument& instrument);
-
-    Status registerInstrument(id_t accountId, const Instrument& instrument);
+    Status registerFeed(id_t accountId, const Feed& feed);
 
     // image uploads
-    Status uploadBackground(id_t accountId, id_t instrumentId,
+    Status uploadBackground(id_t accountId, id_t feedId,
                               const timestamp_t& timestamp, const Payload& payload);
 
-    Status uploadTapestry(id_t accountId, id_t instrumentId,
-                            const timestamp_t& eventTimestamp, const Payload& payload,
-                            const std::string& type = "SUMMARY_PORTRAIT");
-
-    Status uploadLiveTile(id_t accountId, id_t instrumentId,
-                            const timestamp_t& eventTimestamp, const Payload& payload);
-
-    Status uploadObjectStream(id_t accountId, id_t instrumentId,
+    Status uploadObjectStream(id_t accountId, id_t feedId,
```

Comment:
Pls rename ObjectStream to ObjectSnapshot or just Snapshot

Reply:
I would suggest staying with ObjectSnapshot* naming for public methods. Just in case we have any other snapshots.